conda/cli/main_search.py

```diff
@@ -25,6 +27,29 @@ def execute(args, parser):
     else:
         subdirs = context.subdirs
 
+    if args.envs:
+        with spinner("Searching environments for %s" % spec,
+                     not context.verbosity and not context.quiet,
+                     context.json):
+            prefix_matches = search_all_prefixes(spec)
+        formatted_result = tuple({
+            'location': prefix,
+            'package_refs': tuple(PackageRef.from_objects(prefix_rec)
```

Comment:
Should `package_ref` be sorted? E.g., `sorted(..., key=lambda pkg_ref: pkg_ref._pkey)` (no idea if correct, only skimmed that `PackageRef` code.)

Reply:
Yeah I think so.  There are two places it could be done.  Here as your propose, or in `search_all_prefixes()`.  I think as proposed is more consistent with other parts of the code.